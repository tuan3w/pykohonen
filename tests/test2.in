10 10 0.1 2 100
10 10 0.1 4 100
10 10 0.1 6 100
10 10 0.5 2 100
10 10 0.5 4 100
10 10 0.5 6 100
10 10 0.9 2 100
10 10 0.9 4 100
10 10 0.9 6 100
15 15 0.1 2 100
15 15 0.1 4 100
15 15 0.1 6 100
15 15 0.5 2 100
15 15 0.5 4 100
15 15 0.5 6 100
15 15 0.9 2 100
15 15 0.9 4 100
15 15 0.9 6 100
20 20 0.1 2 100
20 20 0.1 4 100
20 20 0.1 6 100
20 20 0.5 2 100
20 20 0.5 4 100
20 20 0.5 6 100
20 20 0.9 2 100
20 20 0.9 4 100
20 20 0.9 6 100
0 0 0 0 0
