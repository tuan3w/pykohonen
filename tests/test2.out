O acerto para confinado foi: 97.9094076655
O acerto para nao confinado foi: 5.71428571429
O acerto total da RNA foi: 87.8881987578



O acerto para confinado foi: 58.5365853659
O acerto para nao confinado foi: 28.5714285714
O acerto total da RNA foi: 55.2795031056



O acerto para confinado foi: 100.0
O acerto para nao confinado foi: 0.0
O acerto total da RNA foi: 89.1304347826



O acerto para confinado foi: 60.6271777003
O acerto para nao confinado foi: 65.7142857143
O acerto total da RNA foi: 61.1801242236



O acerto para confinado foi: 100.0
O acerto para nao confinado foi: 0.0
O acerto total da RNA foi: 89.1304347826



O acerto para confinado foi: 59.2334494774
O acerto para nao confinado foi: 20.0
O acerto total da RNA foi: 54.9689440994



O acerto para confinado foi: 65.5052264808
O acerto para nao confinado foi: 34.2857142857
O acerto total da RNA foi: 62.1118012422



O acerto para confinado foi: 41.1149825784
O acerto para nao confinado foi: 34.2857142857
O acerto total da RNA foi: 40.3726708075



O acerto para confinado foi: 23.6933797909
O acerto para nao confinado foi: 91.4285714286
O acerto total da RNA foi: 31.0559006211



O acerto para confinado foi: 89.1986062718
O acerto para nao confinado foi: 28.5714285714
O acerto total da RNA foi: 82.6086956522



O acerto para confinado foi: 93.7282229965
O acerto para nao confinado foi: 11.4285714286
O acerto total da RNA foi: 84.7826086957



O acerto para confinado foi: 58.1881533101
O acerto para nao confinado foi: 40.0
O acerto total da RNA foi: 56.2111801242



O acerto para confinado foi: 63.4146341463
O acerto para nao confinado foi: 62.8571428571
O acerto total da RNA foi: 63.3540372671



O acerto para confinado foi: 78.0487804878
O acerto para nao confinado foi: 28.5714285714
O acerto total da RNA foi: 72.6708074534



O acerto para confinado foi: 95.818815331
O acerto para nao confinado foi: 5.71428571429
O acerto total da RNA foi: 86.0248447205



O acerto para confinado foi: 82.9268292683
O acerto para nao confinado foi: 22.8571428571
O acerto total da RNA foi: 76.397515528



O acerto para confinado foi: 81.1846689895
O acerto para nao confinado foi: 31.4285714286
O acerto total da RNA foi: 75.7763975155



O acerto para confinado foi: 85.3658536585
O acerto para nao confinado foi: 31.4285714286
O acerto total da RNA foi: 79.5031055901



O acerto para confinado foi: 88.1533101045
O acerto para nao confinado foi: 25.7142857143
O acerto total da RNA foi: 81.3664596273



O acerto para confinado foi: 74.5644599303
O acerto para nao confinado foi: 40.0
O acerto total da RNA foi: 70.8074534161



O acerto para confinado foi: 87.8048780488
O acerto para nao confinado foi: 17.1428571429
O acerto total da RNA foi: 80.1242236025



O acerto para confinado foi: 90.9407665505
O acerto para nao confinado foi: 20.0
O acerto total da RNA foi: 83.2298136646



O acerto para confinado foi: 77.3519163763
O acerto para nao confinado foi: 31.4285714286
O acerto total da RNA foi: 72.3602484472



O acerto para confinado foi: 99.6515679443
O acerto para nao confinado foi: 2.85714285714
O acerto total da RNA foi: 89.1304347826



O acerto para confinado foi: 90.243902439
O acerto para nao confinado foi: 17.1428571429
O acerto total da RNA foi: 82.298136646



O acerto para confinado foi: 69.3379790941
O acerto para nao confinado foi: 48.5714285714
O acerto total da RNA foi: 67.0807453416



O acerto para confinado foi: 81.1846689895
O acerto para nao confinado foi: 34.2857142857
O acerto total da RNA foi: 76.0869565217



